cluster-state-service/main.go

```diff
@@ -36,15 +36,15 @@ func main() {
 		fmt.Printf("Could not initialize logger: %+v", err)
 	}
 	if err := cmd.RootCmd.Execute(); err != nil {
-		log.Criticalf("Error executing: %v", err)
+		log.Criticalf("Error executing: %+v", err)
```

Comment:
As a golang nube, afaik, the '+' adds field names for structs. Whats the advantage here?